src/libsync/filesystem.cpp

```diff
@@ -189,5 +193,10 @@ bool FileSystem::getInode(const QString &filename, quint64 *inode)
     return false;
 }
 
+bool FileSystem::setFolderPermissions(const QString &path)
+{
+
```

Comment:
@mgallien do you need this empty function here? It is not used anymore (also check my comment for the.h file)